src/Leaflet.js

```diff
@@ -30,7 +31,7 @@ export {Class} from './core/Class';
 
 import {Evented} from './core/Events';
 export {Evented};
-export var Mixin = {Events: Evented.prototype};
+// export var Mixin = {Events: Evented.prototype};

```

Comment:
@mourner I was not aware this was a backwards-compat issue, sorry 'bout that :-(